src/Analyzers/CSharp/Tests/UseCollectionInitializer/UseCollectionInitializerTests.cs

```diff
@@ -462,18 +462,68 @@ void M()
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseCollectionInitializer)]
-        public async Task TestMissingWithExistingInitializer()
+        [WorkItem(39146, "https://github.com/dotnet/roslyn/issues/39146")]
+        public async Task TestWithExistingInitializer()
         {
-            await TestMissingInRegularAndScriptAsync(
+            await TestInRegularAndScript1Async(
+@"using System.Collections.Generic;
+
+class C
+{
+    void M()
+    {
+        var c = [||]new List<int>()
```

Comment:
😕 Is this the span of the diagnostic produced?

Reply:
Can we use the new test framework for this test, even if we don't update the rest of the file? It's hard to see how this is appearing.

Reply:
@CyrusNajmabadi @sharwell I had some work for converting this in https://github.com/Youssef1313/roslyn/commit/2e638fde7af58e82dd3e0cf65b37b99e7db337f1. Feel free to cherry-pick and continue on top of it. I don't honestly remember what was remaining for it.